docs/examples/longhorn.md

```diff
@@ -0,0 +1,134 @@
+<!--
+SPDX-FileCopyrightText: 2025 k0s authors
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Longhorn
+
+This tutorial covers installing Longhorn as a Helm extension on k0s. Longhorn provides distributed block storage for Kubernetes with features like snapshots, backup, and live upgrades.
+
+## Prerequisites
```

Comment:
I've also tried to run this on Alpine workers:

```sh
doas apk add open-iscsi
doas rc-update add iscsid default
doas rc-service iscsid start
```

However, that failed completely:

```console
$ kubectl -n longhorn-system get events | grep Error | head -n1
102s        Warning   Failed              pod/longhorn-manager-4n888                     Error: failed to generate container "5e66ceb5e441db848f598691d81bf9bc8c46fa282783af0c6b8ff56d166485e7" spec: failed to generate spec: path "/var/lib/longhorn/" is mounted on "/" but it is not a shared mount
```

Reply:
@twz123 I think it's an issue with the host mount missing options `rshared`. It's probably similar to [Talos](https://longhorn.io/docs/1.10.1/advanced-resources/os-distro-specific/talos-linux-support/).